frontend/src/components/posts/FoodPost.jsx

```diff
@@ -0,0 +1,23 @@
+//react core
```

Comment:
FoodPost is the same as the PostPage or they are separate?

Reply:
Same here. File name is FoodPost.jsx. Function mae is Front (what does it mean btw?).
And you return a div with className wrapper...